packages/ag-charts-react/package.json

```diff
@@ -33,22 +33,16 @@
     "ag-charts-community": "9.3.1-beta.20240609",
     "@types/react": "^18",
     "@types/react-dom": "^18",
-    "babel-preset-react-app": "^9.0.1",
-    "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.14.0",
+    "eslint-plugin-react-hooks": "^4.6.2",
     "react": "^16.9.0",
-    "react-app-polyfill": "^1.0.2",
     "react-dom": "^16.9.0",
-    "serialize-javascript": "2.1.0",
     "tslib": "^1.10.0",
     "typescript": "^5.4.5",
-    "vinyl-source-stream": "^2.0.0",
     "rimraf": "3.0.2"
   },
   "peerDependencies": {
     "ag-charts-community": "9.3.1-beta.20240609",
-    "react": "^16.3.0 || ^17.0.0 || ^18.0.0",
-    "react-dom": "^16.3.0 || ^17.0.0 || ^18.0.0"
+    "react": "^16.8.0 || ^17.0.0 || ^18.0.0 || ^19.0.0"
```

Comment:
We need to stay in alignment with Grid on these:
https://github.com/ag-grid/ag-grid/blob/latest/packages/ag-grid-react/package.json#L46